src/apps/ocioconvert/main.cpp

```diff
@@ -513,22 +551,53 @@ int main(int argc, const char **argv)
             oglApp->setShader(shaderDesc);
             oglApp->reshape(imgwidth, imgheight);
             oglApp->redisplay();
-            oglApp->readImage((float *)img.getBuffer());
+            oglApp->readImage((float *)imgInput.getBuffer());
         }
         else
 #endif // OCIO_GPU_ENABLED
         {
-            const OCIO::BitDepth bitDepth = OCIO::GetBitDepth(spec);
+            // Copy specInput into specOutput.
+            specOutput = specInput;
```

Comment:
Only seeing this now, is `specOutput` correctly initialised when taking the GPU path (`--gpu`)?